### PR TITLE
Fixes air runtime and adds air flow blocker for mappers

### PR DESCRIPTION
**Pull request by AnturK**
Mappers. Not even once.

### CODE REVIEW

File: code/modules/mining/mine_items.dm [hunk 1]
```diff
@@ -454,6 +454,12 @@
 	. = ..()
 	T.air_update_turf(1)
 
+//Inivisible, indestrucctible fans
```
Comment (RemieRichards) [inline code/modules/mining/mine_items.dm]: cc

File: code/modules/mining/mine_items.dm [hunk 2]
```diff
@@ -454,6 +454,12 @@
 	. = ..()
 	T.air_update_turf(1)
 
+//Inivisible, indestructible fans
+/obj/structure/fans/tiny/invisible
+	name = "air flow blocker"
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	invisibility = INVISIBILITY_ABSTRACT
```
Comment (xxalpha) [inline code/modules/mining/mine_items.dm]: anchored?

**Comment by RemieRichards** [comment]
Don't worry @xxalpha I won't tell anyone ;^)

File: code/modules/mining/mine_items.dm [hunk 3]
```diff
@@ -454,6 +454,12 @@
 	. = ..()
 	T.air_update_turf(1)
 
+//Inivisible, indestructible fans
+/obj/structure/fans/tiny/invisible
```
Comment (coiax) [inline code/modules/mining/mine_items.dm]: reeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

Comment (xxalpha) [inline code/modules/mining/mine_items.dm]: :octopus:  what's the matter frog

Comment (coiax) [inline code/modules/mining/mine_items.dm]: I HATE TINY FANS

Comment (AnturK) [inline code/modules/mining/mine_items.dm]: This is better than mappers changing variables they shouldn't, and i see no need to create another type to do exactly the same thing.

**Pull request merged**